### DISCUSSION TITLE
Welcome to OpenCart Discussions!

### DISCUSSION THREAD

**Discussion by danielkerr** [opencart/opencart#10225] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bhavhirani** [comment]
**👋Hello @danielkerr and all the other members,**

I am Bhavik Hirani, and I am also a [Gold Partner of OpenCart](https://bit.ly/ocpartner).
One thing I would like to suggest is that we have already added a lot of new features to OpenCart,
But can we also focus on the Site speed, SEO and many more new features?

### Below are several points we can keep in mind for future versions.

- Add MySQL Indexes (We can add DB index for better search and better query execution)
- Default Time Zone (If the site is used in different countries and the server is in a different country, we can set the default time zone for admin data time)
- Optimize Database Table (With more database usage, we can optimize the database for better speed)
- Add Robots.txt (This will help a lot for search engine crawlers and SEO)
- Import/export product (We need to add this feature for better product listing. This is a basic requirement of every eCommerce site)
- Custom fields for product 

> I am also ready to contribute if you would like to add these new points to OpenCart.
> Please stay in touch with us.

Regards,
Bhavik Hirani.

**Reply by felipoantonoff** [reply]
I liked the ideas are very interesting for future improvements in Opencart that I already consider the best E-commerce platform in general.

I think it's excellent that Opencart remains simple and performs very well, focusing on the core and being 100% modular, unlike other heavy and slow E-commerce.

**Comment by anytizer** [comment]
**Default Time Zone** - Currently sets the time zone twice :-( Profiler shows 2 queries.

**Add MySQL Indexes (We can add DB index for better search and better query execution)** - Would be very nice to have, but as an extra option only, to keep the db creation process simple, by the installer.
